src/SamplesApp/SamplesApp.UITests/MessageDialogTests/MessageDialogTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.MessageDialogTests
+{
+	[TestFixture]
+	public partial class MessageDialogTest : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public void When_Click_Outside_Dialog_Expect_No_Dismiss()
+		{
+			Run("UITests.MessageDialog.MessageDialogTest");
+			var button = _app.Marked("showDialogBtn");
+			_app.WaitForElement(button);
+			button.Tap();
+
+			using var screenshot = TakeScreenshot("BeforeClicking");
+
+			// TODO: Click outside.
```

Comment:
A common trick here is to put an element on the page that you know will be outside the bounds of the dialog, and then tap that element.